sdk/python/kfp/_auth.py

```diff
@@ -34,7 +38,41 @@ def get_gcp_access_token():
     with os.popen('gcloud auth print-access-token') as token:
         return token.read().rstrip()
 
-def get_auth_token(client_id):
+def get_auth_token(client_id, other_client_id, other_client_secret):
+    """Gets auth token from default service account or user account."""
+    if os.path.exists(LOCAL_KFP_CREDENTIAL):
+        # fetch IAP auth token using the locally stored credentials.
+        with open(LOCAL_KFP_CREDENTIAL, 'r') as f:
+            credentials = json.load(f)
+        if client_id in credentials:
+            return id_token_from_refresh_token(credentials[client_id]['other_client_id'],
+                                               credentials[client_id]['other_client_secret'],
+                                               credentials[client_id]['refresh_token'],
```

Comment:
Let's add TODO to later handle the case when token has expired.

Reply:
SG

Reply:
Done.